internal/generator/fluentd/source/tags.go

```diff
@@ -3,7 +3,7 @@ package source
 const (
 	ApplicationTags    = "kubernetes.**"
 	JournalTags        = "journal.** system.var.log**"
-	InfraContainerTags = "**_default_** **_kube-*_** **_openshift-*_** **_openshift_**"
+	InfraContainerTags = "kubernetes.var.log.pods.openshift-*_** **_default_** **_kube-*_**"
```

Comment:
the patterns `**_default_**` and `**_kube-*_**` will not match anything
better replace with
```
kubernetes.var.log.pods.default_** kubernetes.var.log.pods.kube-*_**
```